content/articles/2018-07-broom-0-5-0.Rmarkdown

```diff
@@ -0,0 +1,127 @@
+
+In broom 0.5.0 these tests mostly ensure tibble output along with one or two other consistency items. However, stricter versions of these tests are under development for future releases.
+
+We have also migrated to a new template based documentation strategy. Now repeated documentation material lives in `roxygen2` templates and can easily added to a new tidy method. For an example of how this works, see [`R/aer-tidiers.R`](https://github.com/alexpghayes/broom/blob/master/R/aer-tidiers.R).
+
+## Mixed models are moving to `broom.mixed`
+
+As broom's popularity has grown, broom has grown to encompass a fairly broad range of models. Dave and I have little to no experience with many of these models, and while we can fix bugs in the tidying code, we are no longer able to determine what constitutes a reasonable summary for reasonable summary for many of these models.
+
+Our intended solution is to split broom into several packages for tidying model objects. broom will provide tidiers for the most common tidiers (and those in `base` and `stats`), and then domain experts will manage domain specific tidying packages. Currently we're working on a spec for all these sub-packages to implement. With any luck this we'll have a well-written spec to accompany the next release. We'd like all of the domain specific tidying packages to eventually live in tidymodels, so that users can load a bunch of tools all at once with `library(tidymodels)`.
+
+Mixed models tidiers have long been a bit of a mess in broom, and a while back [`broom.mixed`](
+https://github.com/bbolker/broom.mixed) forked off to clean them up. broom.mixed is now a pilot for the larger project of splitting broom into domain specific tidying packages. We anticipate that broom.mixed will makes its way onto CRAN in the next several weeks, which will allow us to deprecate mixed model tidiers in broom 0.7.0. Although these models are not yet deprecated, there is currently no ongoing development work for them. In particular, the tidiers for:
+
+- lme, lme4 and nmle models
+- brms models
+- rstanarm models
+- mcmc objects
+
+are one release away from deprecation, and effectively frozen.
+
+## New suggested workflow
+
+Broom has been around so long that the original recommended workflow when working with multiple models mostly used `dplyr::group_by` combined with `dplyr::do`. Now, we recommend using list-columns and a `nest-map-unnest` workflow. This is in line with the rest of the tidyverse. We have updated the [kmeans](https://broom.tidyverse.org/articles/kmeans.html), [broom and dplyr](https://broom.tidyverse.org/articles/broom_and_dplyr.html) and  [bootstrapping](https://broom.tidyverse.org/articles/bootstrapping.html) vignettes to reflect this. Additional, we've updated the bootstrapping vignette to use [rsample](https://tidymodels.github.io/rsample/) rather than the now-deprecated `bootstrap()` function.
```

Comment:
I think you should start with what's now recommended, and then mention what it's a change from. That way the new workflow is front and center.
Also, add `()` after functions.